### DISCUSSION TITLE
Match all package version in a group

### DISCUSSION THREAD

**Discussion by Wassap124** [renovatebot/renovate#26857] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

recently I encountered problems due to various `@aws-sdk` packages installed in different minor versions.

This issue (and solution) is documented [here](https://github.com/aws/aws-sdk-js-v3/issues/5435#issuecomment-1879862258) - where the solution is to set them all to the same version.

after setting them all to the same version manually, I extended the group [group:aws-sdk-js-v3Monorepo](https://docs.renovatebot.com/presets-group/#groupaws-sdk-js-v3monorepo) in my renovate config.

so now - it indeed updates all the `@aws-sdk` packages at once - which is great!
but my question is - is there any way to make renovate update them all **_to the same version_**?

Thanks

This was the update:

| Package | Change | Age | Adoption | Passing | Confidence |
|---|---|---|---|---|---|
| [@aws-sdk/client-cloudfront](https://togithub.com/aws/aws-sdk-js-v3/tree/main/clients/client-cloudfront) ([source](https://togithub.com/aws/aws-sdk-js-v3/tree/HEAD/clients/client-cloudfront)) | [`3.490.0` -> `3.496.0`](https://renovatebot.com/diffs/npm/@aws-sdk%2fclient-cloudfront/3.490.0/3.496.0) | [![age](https://developer.mend.io/api/mc/badges/age/npm/@aws-sdk%2fclient-cloudfront/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://developer.mend.io/api/mc/badges/adoption/npm/@aws-sdk%2fclient-cloudfront/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://developer.mend.io/api/mc/badges/compatibility/npm/@aws-sdk%2fclient-cloudfront/3.490.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://developer.mend.io/api/mc/badges/confidence/npm/@aws-sdk%2fclient-cloudfront/3.490.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) |
| [@aws-sdk/client-cloudwatch-logs](https://togithub.com/aws/aws-sdk-js-v3/tree/main/clients/client-cloudwatch-logs) ([source](https://togithub.com/aws/aws-sdk-js-v3/tree/HEAD/clients/client-cloudwatch-logs)) | [`3.490.0` -> `3.496.0`](https://renovatebot.com/diffs/npm/@aws-sdk%2fclient-cloudwatch-logs/3.490.0/3.496.0) | [![age](https://developer.mend.io/api/mc/badges/age/npm/@aws-sdk%2fclient-cloudwatch-logs/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://developer.mend.io/api/mc/badges/adoption/npm/@aws-sdk%2fclient-cloudwatch-logs/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://developer.mend.io/api/mc/badges/compatibility/npm/@aws-sdk%2fclient-cloudwatch-logs/3.490.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://developer.mend.io/api/mc/badges/confidence/npm/@aws-sdk%2fclient-cloudwatch-logs/3.490.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) |
| [@aws-sdk/client-cost-explorer](https://togithub.com/aws/aws-sdk-js-v3/tree/main/clients/client-cost-explorer) ([source](https://togithub.com/aws/aws-sdk-js-v3/tree/HEAD/clients/client-cost-explorer)) | [`3.490.0` -> `3.496.0`](https://renovatebot.com/diffs/npm/@aws-sdk%2fclient-cost-explorer/3.490.0/3.496.0) | [![age](https://developer.mend.io/api/mc/badges/age/npm/@aws-sdk%2fclient-cost-explorer/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://developer.mend.io/api/mc/badges/adoption/npm/@aws-sdk%2fclient-cost-explorer/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://developer.mend.io/api/mc/badges/compatibility/npm/@aws-sdk%2fclient-cost-explorer/3.490.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://developer.mend.io/api/mc/badges/confidence/npm/@aws-sdk%2fclient-cost-explorer/3.490.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) |
| [@aws-sdk/client-dynamodb](https://togithub.com/aws/aws-sdk-js-v3/tree/main/clients/client-dynamodb) ([source](https://togithub.com/aws/aws-sdk-js-v3/tree/HEAD/clients/client-dynamodb)) | [`3.478.0` -> `3.496.0`](https://renovatebot.com/diffs/npm/@aws-sdk%2fclient-dynamodb/3.478.0/3.496.0) | [![age](https://developer.mend.io/api/mc/badges/age/npm/@aws-sdk%2fclient-dynamodb/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://developer.mend.io/api/mc/badges/adoption/npm/@aws-sdk%2fclient-dynamodb/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://developer.mend.io/api/mc/badges/compatibility/npm/@aws-sdk%2fclient-dynamodb/3.478.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://developer.mend.io/api/mc/badges/confidence/npm/@aws-sdk%2fclient-dynamodb/3.478.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) |
| [@aws-sdk/client-efs](https://togithub.com/aws/aws-sdk-js-v3/tree/main/clients/client-efs) ([source](https://togithub.com/aws/aws-sdk-js-v3/tree/HEAD/clients/client-efs)) | [`3.490.0` -> `3.496.0`](https://renovatebot.com/diffs/npm/@aws-sdk%2fclient-efs/3.490.0/3.496.0) | [![age](https://developer.mend.io/api/mc/badges/age/npm/@aws-sdk%2fclient-efs/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://developer.mend.io/api/mc/badges/adoption/npm/@aws-sdk%2fclient-efs/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://developer.mend.io/api/mc/badges/compatibility/npm/@aws-sdk%2fclient-efs/3.490.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://developer.mend.io/api/mc/badges/confidence/npm/@aws-sdk%2fclient-efs/3.490.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) |
| [@aws-sdk/client-kms](https://togithub.com/aws/aws-sdk-js-v3/tree/main/clients/client-kms) ([source](https://togithub.com/aws/aws-sdk-js-v3/tree/HEAD/clients/client-kms)) | [`3.496.0` -> `3.498.0`](https://renovatebot.com/diffs/npm/@aws-sdk%2fclient-kms/3.496.0/3.498.0) | [![age](https://developer.mend.io/api/mc/badges/age/npm/@aws-sdk%2fclient-kms/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://developer.mend.io/api/mc/badges/adoption/npm/@aws-sdk%2fclient-kms/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://developer.mend.io/api/mc/badges/compatibility/npm/@aws-sdk%2fclient-kms/3.496.0/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://developer.mend.io/api/mc/badges/confidence/npm/@aws-sdk%2fclient-kms/3.496.0/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) |
| [@aws-sdk/client-rds](https://togithub.com/aws/aws-sdk-js-v3/tree/main/clients/client-rds) ([source](https://togithub.com/aws/aws-sdk-js-v3/tree/HEAD/clients/client-rds)) | [`3.490.0` -> `3.497.0`](https://renovatebot.com/diffs/npm/@aws-sdk%2fclient-rds/3.490.0/3.497.0) | [![age](https://developer.mend.io/api/mc/badges/age/npm/@aws-sdk%2fclient-rds/3.497.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://developer.mend.io/api/mc/badges/adoption/npm/@aws-sdk%2fclient-rds/3.497.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://developer.mend.io/api/mc/badges/compatibility/npm/@aws-sdk%2fclient-rds/3.490.0/3.497.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://developer.mend.io/api/mc/badges/confidence/npm/@aws-sdk%2fclient-rds/3.490.0/3.497.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) |
| [@aws-sdk/client-route-53](https://togithub.com/aws/aws-sdk-js-v3/tree/main/clients/client-route-53) ([source](https://togithub.com/aws/aws-sdk-js-v3/tree/HEAD/clients/client-route-53)) | [`3.490.0` -> `3.496.0`](https://renovatebot.com/diffs/npm/@aws-sdk%2fclient-route-53/3.490.0/3.496.0) | [![age](https://developer.mend.io/api/mc/badges/age/npm/@aws-sdk%2fclient-route-53/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://developer.mend.io/api/mc/badges/adoption/npm/@aws-sdk%2fclient-route-53/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://developer.mend.io/api/mc/badges/compatibility/npm/@aws-sdk%2fclient-route-53/3.490.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://developer.mend.io/api/mc/badges/confidence/npm/@aws-sdk%2fclient-route-53/3.490.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) |
| [@aws-sdk/client-s3](https://togithub.com/aws/aws-sdk-js-v3/tree/main/clients/client-s3) ([source](https://togithub.com/aws/aws-sdk-js-v3/tree/HEAD/clients/client-s3)) | [`3.496.0` -> `3.498.0`](https://renovatebot.com/diffs/npm/@aws-sdk%2fclient-s3/3.496.0/3.498.0) | [![age](https://developer.mend.io/api/mc/badges/age/npm/@aws-sdk%2fclient-s3/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://developer.mend.io/api/mc/badges/adoption/npm/@aws-sdk%2fclient-s3/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://developer.mend.io/api/mc/badges/compatibility/npm/@aws-sdk%2fclient-s3/3.496.0/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://developer.mend.io/api/mc/badges/confidence/npm/@aws-sdk%2fclient-s3/3.496.0/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) |
| [@aws-sdk/client-secrets-manager](https://togithub.com/aws/aws-sdk-js-v3/tree/main/clients/client-secrets-manager) ([source](https://togithub.com/aws/aws-sdk-js-v3/tree/HEAD/clients/client-secrets-manager)) | [`3.480.0` -> `3.496.0`](https://renovatebot.com/diffs/npm/@aws-sdk%2fclient-secrets-manager/3.480.0/3.496.0) | [![age](https://developer.mend.io/api/mc/badges/age/npm/@aws-sdk%2fclient-secrets-manager/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://developer.mend.io/api/mc/badges/adoption/npm/@aws-sdk%2fclient-secrets-manager/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://developer.mend.io/api/mc/badges/compatibility/npm/@aws-sdk%2fclient-secrets-manager/3.480.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://developer.mend.io/api/mc/badges/confidence/npm/@aws-sdk%2fclient-secrets-manager/3.480.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) |
| [@aws-sdk/credential-providers](https://togithub.com/aws/aws-sdk-js-v3/tree/main/packages/credential-providers) ([source](https://togithub.com/aws/aws-sdk-js-v3/tree/HEAD/packages/credential-providers)) | [`3.496.0` -> `3.498.0`](https://renovatebot.com/diffs/npm/@aws-sdk%2fcredential-providers/3.496.0/3.498.0) | [![age](https://developer.mend.io/api/mc/badges/age/npm/@aws-sdk%2fcredential-providers/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://developer.mend.io/api/mc/badges/adoption/npm/@aws-sdk%2fcredential-providers/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://developer.mend.io/api/mc/badges/compatibility/npm/@aws-sdk%2fcredential-providers/3.496.0/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://developer.mend.io/api/mc/badges/confidence/npm/@aws-sdk%2fcredential-providers/3.496.0/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) |
| [@aws-sdk/s3-request-presigner](https://togithub.com/aws/aws-sdk-js-v3/tree/main/packages/s3-request-presigner) ([source](https://togithub.com/aws/aws-sdk-js-v3/tree/HEAD/packages/s3-request-presigner)) | [`3.490.0` -> `3.498.0`](https://renovatebot.com/diffs/npm/@aws-sdk%2fs3-request-presigner/3.490.0/3.498.0) | [![age](https://developer.mend.io/api/mc/badges/age/npm/@aws-sdk%2fs3-request-presigner/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://developer.mend.io/api/mc/badges/adoption/npm/@aws-sdk%2fs3-request-presigner/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://developer.mend.io/api/mc/badges/compatibility/npm/@aws-sdk%2fs3-request-presigner/3.490.0/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://developer.mend.io/api/mc/badges/confidence/npm/@aws-sdk%2fs3-request-presigner/3.490.0/3.498.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) |
| [@aws-sdk/util-dynamodb](https://togithub.com/aws/aws-sdk-js-v3/tree/main/packages/util-dynamodb) ([source](https://togithub.com/aws/aws-sdk-js-v3/tree/HEAD/packages/util-dynamodb)) | [`3.415.0` -> `3.496.0`](https://renovatebot.com/diffs/npm/@aws-sdk%2futil-dynamodb/3.415.0/3.496.0) | [![age](https://developer.mend.io/api/mc/badges/age/npm/@aws-sdk%2futil-dynamodb/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://developer.mend.io/api/mc/badges/adoption/npm/@aws-sdk%2futil-dynamodb/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://developer.mend.io/api/mc/badges/compatibility/npm/@aws-sdk%2futil-dynamodb/3.415.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://developer.mend.io/api/mc/badges/confidence/npm/@aws-sdk%2futil-dynamodb/3.415.0/3.496.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) |

**Comment by rarkins** [comment]
Not currently, and it might be quite a challenge. Do you know why they are spread between 3.496.0, 3.497.0 and 3.498.0? It seems like they have common versions, but they don't always get a release? If so, you may have _no_ version which they all have. In this case it looks like four got updated to 498, one to 497, and the rest to only 496. We don't know if the first five here all have a 496 too or if they maybe skipped it.

**Comment by Wassap124** [comment]
Thanks for the quick reply 🙏 

it seems that not all get deployed with all versions, i checked with [client-rds](https://www.npmjs.com/package/@aws-sdk/client-rds?activeTab=versions) for example, and it has a `499` but not a `498`.

**Reply by rarkins** [reply]
So does that make it an impossible ask for this particular use case?

**Reply by Wassap124** [reply]
i wouldn't say impossible, cause there are at least **_some_** versions which are shared across the board (like we had before renovate created a pr).

**Reply by rarkins** [reply]
So whenever there's not a common version, suppress the updates? (potentially forever if they never all release together again)

**Reply by Wassap124** [reply]
exactly
